app/views/donations/index.html.erb

```diff
@@ -52,12 +52,12 @@
           <% end %>
           <div class="form-group col-lg-3 col-md-4 col-sm-6 col-xs-12">
             <%= label_tag "from Date" %>
-            <%= date_field :dates, :date_from, class: "form-control", value: @date_from %>
+            <%= text_field_tag 'dates[date_from]', @date_from, { id: 'dates_date_from', placeholder: 'mm/dd/YYY', class: 'form-control', 'data-provide': 'datepicker' } %>
```

Comment:
Is there a reason that this had to change from a date_field to a text_field_tag?

Reply:
Yes! Unfortunately the gem bootstrap-datepicker-rails does not support the date_field because of the value. :/